mathjax3-ts/output/chtml/Wrappers/munderover.ts

```diff
@@ -73,8 +80,9 @@ export class CHTMLmunder extends CHTMLmsub {
         const basebox = this.base.getBBox();
         const underbox = this.script.getBBox();
         const [k, v] = this.getUnderKV(basebox, underbox);
+        const delta = DELTA * this.baseCore.bbox.ic / 2;
```

Comment:
delta vs DELTA

Reply:
If you can't use the distinction between upper and lower case, why have them be distinct?  The all-caps version is a constant, and the lower case one is a variable.  I'm not sure why that is such a problem.  In any case, I've changed the `delta` to something less meaningful.